packages/ckeditor5-dev-env/lib/release-tools/utils/transform-commit/transform-commit-utils.js

```diff
@@ -37,31 +37,100 @@ const transformCommitUtils = {
 	},
 
 	/**
-	 * Changes user's name to link that leads to the user's profile.
+	 * Replaces reference to user (@name) with a link to his profile.
 	 *
-	 * @param {String} sentence
+	 * @param {String} comment
 	 * @returns {String}
 	 */
-	linkGithubUsers( sentence ) {
-		return sentence.replace( /@([\w\d_-]+)/g, '[@$1](https://github.com/$1)' );
+	linkToGithubUser( comment ) {
+		return comment.replace( /@([0-9A-Z_-]+\/?)/ig, ( matchText, nickName ) => {
```

Comment:
`matchedText`

Reply:
`/@\w+(?!\/)/g` and you don't need that `if ()` below

Reply:
It does not help. I leave it as it is.